google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/StreamWriter.java

```diff
@@ -272,37 +272,9 @@ private void setException(Throwable t) {
       messagesBatchLock.unlock();
       appendAndRefreshAppendLock.unlock();
     }
-
     return outstandingAppend.appendResult;
   }
 
-  /**
-   * This is the general flush method for asynchronise append operation. When you have outstanding
-   * append requests, calling flush will make sure all outstanding append requests completed and
-   * successful. Otherwise there will be an exception thrown.
-   *
-   * @throws Exception
-   */
-  public void flushAll(long timeoutMillis) throws Exception {
```

Comment:
@yirutang Would you explain whether this removal is a backward compatible change from the library users's perspective?

 (I'm surveying how much CLIRR helps Java libraries to follow semantic versioning or not)

Reply:
As of now I am pretty sure, there is no user on this. We would eventually provide this method but there is still some dispute about this.

Reply:
Got it. Thanks.